plugins/onepad/onepad.h

```diff
@@ -21,8 +21,8 @@
 
 #ifndef __PAD_H__
 #define __PAD_H__
-
-#define GAMEPAD_NUMBER 2 // numbers of gamepad
+//this is where GAMEPAD_NUMBER max is assigned, possibly rewrite later for dynamic based on sdlgamepads detected
```

Comment:
If I'm right this value is the maximum number of PS2 gamepad. So it should be a constant. You could always use a keyboard to emulate a PS2 gamepad.